src/extension/ncwms/src/main/java/au/org/emii/geoserver/wms/LayerDescriptor.java

```diff
@@ -59,6 +65,8 @@ public String getUrlFieldName() {
         return urlField;
     }
 
+    public String getVariableName() { return variable; }
```

Comment:
But since you're only reusing an existing field - up to you whether you want to change it